### PR TITLE
[chore] Error codes cleanup

### PR DESCRIPTION
**Pull request by nebula-aac**
**Notes for Reviewers**

This PR removed some of the unused error codes.

I would create new ones, but it seems like they weren't being used anywhere in the repo.

There is one warning on one error code: `ErrInvalidRequestObjectCode` which does exist, but the `errorutil` doesn't check if the error message format is appending new strings, or if the fields are nil.

I will be adding a check in `meshkit` for it.

If there are any error codes you would like to use here, please let me know, and we can make revisions.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: server/handlers/error.go [hunk 1]
```diff
@@ -113,6 +110,7 @@ const (
 	ErrGetMeshModelsCode                = "2257"
 	ErrGetUserDetailsCode               = "2258"
 	ErrResolvingRelationship            = "2259"
+	ErrGetLatestVersionCode             = ""
```
Comment (Revolyssup) [inline server/handlers/error.go:113]: Why is this empty?

**Review by Revolyssup — CHANGES_REQUESTED** [review]

File: server/handlers/error.go (hunk 1 above)
Comment (nebula-aac) [reply server/handlers/error.go]: @Revolyssup This has been revised to use a random number, until I can clean up the duplicates.

**Comment by leecalcote** [comment]
Refreshing with changes from HEAD. 

@Revolyssup your eyes on this are appreciated.

**Comment by leecalcote** [comment]
@nebula-aac did we end up with new issues to track enhancements on the errorutils? I can't remember...

**Comment by nebula-aac** [comment]
I will create a new issue tracking `errorutil`.

//@leecalcote

**Comment by leecalcote** [comment]
Thank you much. 🙂

**Review by leecalcote — APPROVED** [review]
Lgtm

**Pull request merged**